python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -536,3 +563,150 @@ def to_arrow(self):
                 arrow_polygons,
             ],
         )
+
+    def contains(self, other, align=True):
```

Comment:
Per discussion today, rename to `contains_properly` and update doc below?

Reply:
Done.